tests/lite/test_lite.py

```diff
@@ -164,6 +164,57 @@ def test_setup_dataloaders_return_type():
     assert lite_dataloader1.dataset is dataset1
 
 
+def test_setup_custom_dataloaders():
+    """Test that the setup_dataloaders method returns the dataloaders wrapped as LiteDataLoader."""
+    lite = EmptyLite()
+
+    class CustomDataLoader(DataLoader):
+        def __init__(self, value: int = 2, *args, **kwargs):
+            self.value = value
+            super().__init__(range(value), *args, **kwargs)
```

Comment:
There's no reason for us to add support for this, as this is a bug in the DataLoader implementation.

Adding arbitrary `**kwargs` when opens the door to duplicate `dataset` arguments, but this should be fixed by the user, not by us.

The fix only works for the first argument and if the name is dataset, but this would still break for any of the other arguments